### PR TITLE
Create INS1020EFSmissionsmuseum.xml

### PR DESCRIPTION
**Pull request by karljonaskarlsson**
Created INS for UppEt20

### CODE REVIEW

File: new/INS1020EFSmissionsmuseum.xml
Comment (nafisa-valieva) [inline new/INS1020EFSmissionsmuseum.xml:1]: Nice place!

File: new/INS1020EFSmissionsmuseum.xml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
```
Comment (CarstenHoffmannMarburg) [inline new/INS1020EFSmissionsmuseum.xml:37]: Swedish is missing (cf. title, names).

**Review by CarstenHoffmannMarburg — CHANGES_REQUESTED** [review]

File: new/INS1020EFSmissionsmuseum.xml (hunk 1 above)
Comment (eu-genia) [reply new/INS1020EFSmissionsmuseum.xml:37]: Just to explain better: `langUsage `is a list of languages used in the record (= language abbreviations used in `xml:lang` elements). 

In this case `gez `is not used and therefore not needed.

`sv `= Swedish would become needed if `xml:lang` is added to `placeName`; at the moment it is not specified.

File: new/INS1020EFSmissionsmuseum.xml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+            <change who="JK" when="2024-09-16">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPlace>
+                <place type="museum" subtype="institution">
+                    <placeName n="1">Evangeliska Fosterlands-Stiftelsens etnografiska missionsmuseum</placeName>
+                    <placeName n="2">Evangeliska Fosterlandsstiftelsens etnografiska missionsmuseum</placeName>
+                    <country ref="LOC5829Sweden"></country>
```
Comment (eu-genia) [inline new/INS1020EFSmissionsmuseum.xml:50]: for institutions we always need the settlement(town), as this is then printed with the shelfmark of manuscripts
(which would then become `Stockholm`, Ev...., ...)

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/INS1020EFSmissionsmuseum.xml (hunk 2 above)
Comment (karljonaskarlsson) [reply new/INS1020EFSmissionsmuseum.xml:50]: Thanks, good to know!

File: new/INS1020EFSmissionsmuseum.xml [hunk 3]
```diff
@@ -45,9 +45,10 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         <body>
             <listPlace>
                 <place type="museum" subtype="institution">
-                    <placeName n="1">Evangeliska Fosterlands-Stiftelsens etnografiska missionsmuseum</placeName>
-                    <placeName n="2">Evangeliska Fosterlandsstiftelsens etnografiska missionsmuseum</placeName>
-                    <country ref="LOC5829Sweden"></country>
+                    <placeName n="1" xml:lang="sv">Evangeliska Fosterlands-Stiftelsens etnografiska missionsmuseum</placeName>
+                    <placeName n="2" xml:lang="sv">Evangeliska Fosterlandsstiftelsens etnografiska missionsmuseum</placeName>
+                    <settlement ref="wd:Q1754"/>
+                    <country ref="wd:Q39"/>
```
Comment (eu-genia) [inline new/INS1020EFSmissionsmuseum.xml:51]: I know that the guidelines suggest this possibility but wherever we have a local Bm ID it is better to use it, as then we can better search e.g. if you want to quickly look up for all attestations of Stockholm in the database. At the moment the app is not configured to also look for all attestations of wikidata IDs.

Comment (karljonaskarlsson) [reply new/INS1020EFSmissionsmuseum.xml:51]: Shall I create Stockholm, then? It does not exist yet, as far as I can see :)

Comment (eu-genia) [reply new/INS1020EFSmissionsmuseum.xml:51]: Thank you, yes, with a link to wd

**Review by CarstenHoffmannMarburg — APPROVED** [review]

**Review by CarstenHoffmannMarburg — APPROVED** [review]

**Review by eu-genia — APPROVED** [review]

**Pull request merged**